app/views/insured/families/_qles_carousel.html.erb

```diff
@@ -10,15 +10,18 @@
             <% end %>
           </span>
         </div>
-        <div class="btn-group">
-          <span type="button" class="btn btn-primary<% if @manually_picked_role == 'individual_market_events' %> active <% end %>">
-            <% if controller.action_name == "find_sep" %>
-              <%= link_to "Individual", find_sep_insured_families_path(shop_for_plan: "shop_for_plan", market: "individual_market_events") %>
-            <% else %>
-              <%= link_to "Individual", family_account_path(tab: 'home', market: "individual_market_events") %>
-            <% end %> 
-          </span>
-        </div>
+
+        <% unless @person.active_employee_roles.present?%>
```

Comment:
Not going to executed since we are checking @multi_roles before. we can remove it but As per ticket I just disabled.